packages/nvidia-container-toolkit/nvidia-container-toolkit-config-k8s

```diff
@@ -12,3 +12,12 @@ root = "/"
 path = "/usr/bin/nvidia-container-cli"
 environment = []
 ldconfig = "@/sbin/ldconfig"
+
+[nvidia-container-runtime]
```

Comment:
Posting this here for the lack of a better place to post it:

You can just say

```
nvidia-container-toolkit: support both legacy and CDI stacks

Update the NVIDIA Container Toolkit configuration template to dynamically
select the mode based off the configurations used in the Kubernetes Device
Plugin.
```